scripts/run-live-build.sh

```diff
@@ -93,14 +93,25 @@ if [[ "$RUN_TYPE" == "$ALL_RUN_TYPE" || "$RUN_TYPE" == "$VM_RUN_TYPE" ]]; then
 	cp -r "$TOP/live-build/config/hooks/$VM_RUN_TYPE/." "$build_dir/config/hooks"
 fi
 
+sed "s/@@PLATFORM@@/$APPLIANCE_PLATFORM/" \
+	<"$build_dir/config/package-lists/delphix-platform.list.chroot.in" \
+	>"$build_dir/config/package-lists/delphix-platform.list.chroot"
+
+if [[ -d "$TOP/live-build/variants/$APPLIANCE_VARIANT/package-lists" ]]; then
+	for list in "$TOP/live-build/variants/$APPLIANCE_VARIANT/package-lists/"*; do
```

Comment:
nit: you can remove the `-d` check, and set `shopt -s nullglob`, which will produce an empty list if directory doesn't exist or is empty.